app/models/report.rb

```diff
@@ -37,6 +37,19 @@ def activity_must_be_a_fund
     end
   end
 
+  def next_four_financial_quarters
```

Comment:
Yeah, we should eventually move this into a `FinancialQuarter` class (value object?) perhaps, but this is good for now :)